backend/src/user.ts

```diff
@@ -0,0 +1,68 @@
+import { Path, GET, PathParam, POST, PUT, DELETE } from "typescript-rest";
+
+interface User {
+    id: number;
+    username: string;
+    firstName: string;
+    lastName: string;
+    email: string;
+    phone: string;
+    userStatus: string;
+};
+
+@Path("/user")
+export class UserService {
+    @POST
+    createUser() {
+        // create User and return its info
```

Comment:
comment isn't needed anymore

Reply:
Well, right now it's all just for mocking the API, so I'd just keep it for now so it's clear what this function should do once it has actual functionality.